src/Home/ActivationStatus.tsx

```diff
@@ -26,19 +26,26 @@ export const ActivationStatus: FunctionComponent<ActivationStatusProps> = ({
   const { t } = useTranslation()
 
   const bodyText = isActive ? t("common.enabled") : t("common.disabled")
-  const icon = isActive ? Icons.CheckInCircle : Icons.XInCircle
-  const iconFill = isActive ? Colors.success100 : Colors.danger75
+  const leftIcon = isActive ? Icons.CheckInCircle : Icons.XInCircle
```

Comment:
Looking for thoughts: is this big enough that it should become a config object or is that overkill?

Reply:
Maybe it could become a constant? you could declare the `activeAssets` and `inactiveAssets`?  And then use it? but you would have to put thems somewhere and this component is not too big? Maybe it's worth separating? It's also not symmetrical(both left and right) so it might be ok to leave as is. Your call